geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/hash/AbstractHScanIntegrationTest.java

```diff
@@ -121,17 +156,10 @@ public void givenCount_whenCountParameterIsNegative_returnsSyntaxError() {
   }
 
   @Test
-  public void givenMultipleCounts_whenAnyCountParameterIsNotAnInteger_returnsNotIntegerError() {
-    jedis.hset("a", "b", "1");
-    assertThatThrownBy(() -> jedis.sendCommand(Protocol.Command.HSCAN, "a", "0", "COUNT", "3",
-        "COUNT", "sjlfs", "COUNT", "1"))
-            .hasMessageContaining(ERROR_NOT_INTEGER);
-  }
+  public void givenMultipleCounts_whenAnyCountParameterIsLessThanOne_DoesNotError() {
+    jedis.hset("key", "b", "1");
```

Comment:
The title of this test seems incorrect.  When any count parameter is less than 0, we return a syntax error.  maybe instead of saying `DoesNotError` we could change it back to `returnsSyntaxError`

Reply:
will do